test/lib/data_dog_report_test.rb

```diff
@@ -6,7 +6,7 @@ class DataDogReportTest < ActiveSupport::TestCase
   it 'must send a create_event request to Datadog' do
     VCR.use_cassette('datadog_error_request') do
       response = DataDogReport.error('This is an error sample')
-      _(response.to_hash[:status]).must_equal 'ok'
+      _(response).must_equal nil
```

Comment:
This is a strange test.  Why did this change from 200 status to no status?

Reply:
Find a way to the **puts** message to stdout instead. No need to check that response is nil.